src/main/scala/org/phenoscape/owlery/Knowledgebase.scala

```diff
@@ -25,25 +25,7 @@ case class Knowledgebase(name: String, reasoner: OWLReasoner) {
 
   private val factory = OWLManager.getOWLDataFactory
   private lazy val owlet = new Owlet(this.reasoner)
-  private val jsonldContext = Map(
-    "@context" -> Map(
-      "subClassOf" -> Map(
-        "@id" -> RDFS.subClassOf.getURI,
-        "@type" -> "@id"),
-      "superClassOf" -> Map(
-        "@reverse" -> RDFS.subClassOf.getURI,
-        "@type" -> "@id"),
-      "equivalentClass" -> Map(
-        "@id" -> OWL2.equivalentClass.getURI,
-        "@type" -> "@id"),
-      "hasInstance" -> Map(
-        "@reverse" -> RDF.`type`.getURI,
-        "@type" -> "@id"),
-      "value" -> Map(
-        "@id" -> RDF.value.getURI),
-      "isSatisfiable" -> Map(
-        "@id" -> Vocabulary.isSatisfiable,
-        "@type" -> "http://www.w3.org/2001/XMLSchema#boolean"))).toJson
+  private val jsonldContext = Map("@context" -> "https://raw.githubusercontent.com/phenoscape/owlery/master/context.jsonld").toJson
```

Comment:
I think we should serve that through Github pages. I suppose there isn't a project Github Pages site yet, but it'd be easy to add, either under `docs/` on master, or `gh-pages` branch. I think `docs/` would be preferable, because then changes to the `context.jsonld` file can be managed as part of pull requests against the master branch.

Reply:
> Is it that the URL may be more stable?

Yes, it's more "canonical". Specifically, it doesn't have the implementation (namely coming out of a Github repo through the specific mechanism that Github currently uses to accomplish that ) encoded into it, and hence is robust to changing in the future the exact mechanism to serving it. (It would be at https://owlery.phenoscape.org/json/context.jsonld, for example.)